sos/plugins/block.py

```diff
@@ -32,6 +41,9 @@ def setup(self):
             "ls -lanR /sys/block"
         ])
 
+        if self.get_option("all"):
```

Comment:
If this is a fast thing to do why not just run it regardless and not make it optionable? I tend to think the slow tasks should be optional and everything else run by default.